src/remote_gym/remote_environment.py

```diff
@@ -236,7 +244,15 @@ def create_channel_credentials() -> grpc.ChannelCredentials:
         client_credentials = create_channel_credentials()
         connection = dm_env_rpc_connection.create_secure_channel_and_connect(server_address, client_credentials)
         remote_environment, _ = dm_env_adaptor.create_and_join_world(
-            connection, create_world_settings={}, join_world_settings={}
+            connection,
+            create_world_settings={
+                "args": json.dumps(
+                    OmegaConf.to_container(self.remote_args)
```

Comment:
I dislike this, but wasn't able to pull that into where the omegaconf is passed in the first place. Imo, we should be able to safely assume a dict here.

Reply:
If the typing here is really DictConfig and not a dict, this should be reflected at the very top in the class instantiation arguments.

Currently it's `remote_args: dict = {}`.

Reply:
But I agree with you, we should be able so assume it's a dict. Is there no way to fix this in the place it is passed?

Reply:
> If the typing here is really DictConfig
It originates from the config/config_reading, however, I believe its safe to convert before passing params. Will append that to the respective PR and remove Omegaconf here.

Reply:
Yeah, `params` of EnvironmentConfig is just of type dict and not touched by OmegaConf. It should be safe to convert back.